src/App.tsx

```diff
@@ -1,24 +1,139 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
+import * as postService from './api/todos';
+import { getFilteredTodos } from './utils/functions';
+import { TodoFilter } from './components/TodoFilter';
+import { Header, USER_ID } from './components/Header';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Status } from './types/Status';
+import { ErrorMessages } from './utils/ErrorMessages';
+import { TodoList } from './components/TodoList';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+const initialTodos: Todo[] = [];
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todoList, setTodoList] = useState<Todo[]>(initialTodos);
+  const [filterBy, setFilterBy] = useState(Status.All);
+  const [errorMessage, setErrorMessage] = useState(ErrorMessages.NoError);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingId, setLoadingId] = useState<number[]>([]);
+
+  const filteredTodos = getFilteredTodos(todoList, filterBy);
+
+  useEffect(() => {
+    postService.getTodos(USER_ID)
+      .then(setTodoList)
+      .catch(() => {
+        setErrorMessage(ErrorMessages.LoadError);
+        throw new Error();
```

Comment:
The error text must be passed to the constructor of the `Error` class, otherwise it makes little sense.
Fix this in all code.